stagemonitor-core/src/main/java/org/stagemonitor/core/StagemonitorCoreConfigurationSourceInitializer.java

```diff
@@ -76,19 +76,19 @@ private void assertElasticsearchIsAvailable(ElasticsearchClient elasticsearchCli
 	}
 
 	/**
-	 * Creates and registers a SpringCloudConfigConfigurationSource for each configuration url
+	 * Creates and registers a RemotePropertiesConfigurationSource for each configuration url
 	 */
-	private void addSpringCloudConfigurationSources(ConfigurationRegistry configuration, CorePlugin corePlugin) {
+	private void addRemotePropertiesConfigurationSources(ConfigurationRegistry configuration, CorePlugin corePlugin) {
 		// Validating necessary properties
-		final List<String> configurationUrls = new ArrayList<String>(corePlugin.getRemotePropertiesConfigUrls());
+		final ArrayList<URL> configurationUrls = new ArrayList<URL>(corePlugin.getRemotePropertiesConfigUrls());
```

Comment:
As getRemotePropertiesConfigUrls() already returns a list now, you don't have to create a new array list.

Reply:
Refactored